test/versioned/nestjs/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "nestjs-tests",
+  "version": "0.0.0",
+  "private": true,
+  "tests": [
+    {
+      "engines": {
+        "node": ">=14"
+      },
+      "dependencies": {
+        "@nestjs/cli": ">=8.0.0"
```

Comment:
i'll allow @jmartin4563 to finish reviewing but does instrumentation crash if it is less than 8? If so i would do a semver check in the instrumentation

Reply:
Yeah, I tried 7.0.0 and it failed. `nest build` fails for me with typescript errors, so I'm not sure if it's even supported anymore or not. The least 7.x.x release was in November 2020, so it seems reasonable to drop support for it from our side.